app/views/branches/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'shared/notice' %>
+<div class="container mt-4">
+  <div class="border p-4">
+    <div class="d-flex justify-content-between align-items-center mb-4">
+      <h2>Branches</h2>
```

Comment:
check for branches and show no branches found, same as earlier

Reply:
Done